### DISCUSSION TITLE
Empty following feed

### DISCUSSION THREAD

**Discussion by NotaInutilis** [community/community#60923] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi,
I've got a problem with the Following feed on the dashboard. It's desperately empty, even with dozens of starred repos I'd like to see updates on.

![Screenshot 2023-07-15 at 15-43-06 Build software better together](https://github.com/community/community/assets/9675966/cb5f1533-0b26-4e41-bfc9-dfbd8e7d29c1)

I've found other reports of the issue but these seems to not have answers from GitHub staff able to fix this so I'm opening another one.

Thanks for the help,
Cheers!

**Comment by GhostOf0days** [comment]
The following feed consists of people doing something on your repo (starring it, making pull requests, making commits, etc.), people starting to follow you, as well as activity from people you follow. If no has done anything on any of your repos or no one has started to follow you, it's empty because the people you follow haven't done anything recently (your case as you follow one account) or you follow no one.

**Reply by JusperLee** [reply]
I only have two people on my account and I can't find any of my previous activity.
